specification/compute/resource-manager/Microsoft.Compute/stable/2021-08-01/disk.json

```diff
@@ -2935,6 +2978,18 @@
           {
             "value": "TrustedLaunch",
             "description": "Trusted Launch provides security features such as secure boot and virtual Trusted Platform Module (vTPM)"
+          },
+          {
+            "value": "ConfidentialVM_VMGuestStateOnlyEncryptedWithPlatformKey",
+            "description": "Indicates Confidential VM disk with only VM guest state blob encrypted"
+          },
+          {
+            "value": "ConfidentialVM_DiskEncryptedWithPlatformKey",
+            "description": "Indicates Confidential VM disk with both OS disk blob and VM guest state blob encrypted with a platform managed key"
```

Comment:
don't mention blobs.. both Os disk and VM guest state encrypted

Reply:
same for all other additions.. remove any references to blob

Reply:
removed blob references:

also modified following descriptions -


{
                 "value": "ImportSecure",
-                "description": "Similar to Import create option. Create a new Trusted Launch VM or Confidential VM supported disk by importing additional VM guest state blob specified by securityDataUri in storage account specified by storageAccountId"
+                "description": "Similar to Import create option. Create a new Trusted Launch VM or Confidential VM supported disk by importing additional blob for VM guest state specified by securityDataUri in storage account specified by storageAccountId"
               },


"securityDataUri": {
           "type": "string",
-          "description": "If createOption is ImportSecure, this is the URI of a VM guest state blob to be imported."
+          "description": "If createOption is ImportSecure, this is the URI of a blob to be imported into VM guest state."
         }